pymongo/mongo_client.py

```diff
@@ -368,45 +369,57 @@ def __init__(
 
           | **SSL configuration:**
 
-          - `ssl`: If ``True``, create the connection to the server using SSL.
-            Defaults to ``False``.
-          - `ssl_certfile`: The certificate file used to identify the local
-            connection against mongod. Implies ``ssl=True``. Defaults to
-            ``None``.
-          - `ssl_keyfile`: The private keyfile used to identify the local
-            connection against mongod.  If included with the ``certfile`` then
-            only the ``ssl_certfile`` is needed.  Implies ``ssl=True``.
-            Defaults to ``None``.
-          - `ssl_pem_passphrase`: The password or passphrase for decrypting
-            the private key in ``ssl_certfile`` or ``ssl_keyfile``. Only
-            necessary if the private key is encrypted. Only supported by python
-            2.7.9+ (pypy 2.5.1+) and 3.3+. Defaults to ``None``.
-          - `ssl_cert_reqs`: Specifies whether a certificate is required from
-            the other side of the connection, and whether it will be validated
-            if provided. It must be one of the three values ``ssl.CERT_NONE``
-            (certificates ignored), ``ssl.CERT_REQUIRED`` (certificates
-            required and validated), or ``ssl.CERT_OPTIONAL`` (the same as
-            CERT_REQUIRED, unless the server was configured to use anonymous
-            ciphers). If the value of this parameter is not ``ssl.CERT_NONE``
-            and a value is not provided for ``ssl_ca_certs`` PyMongo will
-            attempt to load system provided CA certificates. If the python
-            version in use does not support loading system CA certificates
-            then the ``ssl_ca_certs`` parameter must point to a file of CA
-            certificates. Implies ``ssl=True``. Defaults to
-            ``ssl.CERT_REQUIRED`` if not provided and ``ssl=True``.
-          - `ssl_ca_certs`: The ca_certs file contains a set of concatenated
+          - `tls`: (boolean) If ``True``, create the connection to the server
+            using TLS. Defaults to ``False``.
+          - `ssl`: (boolean) Alias for ``tls``.
+          - `tlsInsecure`: (boolean) Specify whether TLS constraints should be
+            relaxed as much as possible. Setting ``tlsInsecure=True`` implies
+            ``tlsAllowInvalidCertificates=False`` and
+            ``tlsAllowInvalidHostnames=False``. Defaults to ``False``. Think
+            very carefully before setting this to ``True`` as it could allow
+            malicious third-parties to exploit TLS vulnerabilities to attack
```

Comment:
It's not about vulnerabilities. Say "...setting this to ``True`` dramatically reduces the security of TLS..."